azurelinuxagent/ga/monitor.py

```diff
@@ -406,7 +406,11 @@ def init_cgroups():
             CGroupsTelemetry.track_cgroup(CGroups.for_extension(""))
             CGroupsTelemetry.track_agent()
         except Exception as e:
-            logger.error("monitor: Exception tracking wrapper and agent: {0} [{1}]", e, traceback.format_exc())
+            # when a hierarchy is not mounted, we raise an exception
+            # and we should therefore only issue a warning, since this
+            # is not unexpected
+            logger.warn("Monitor: cgroups not initialized: {0}", ustr(e))
```

Comment:
Why do you need the monitor prefix? I thought there was a previous change to make that clearer in the logs already?

Reply:
The monitor and environment threads do not use the logging prefix as things stand

Reply:
They should.